src/components/data/EntitiesContent.js

```diff
@@ -468,10 +469,14 @@ const EntitiesContent = ({
           selected: selectedEntities,
           setSelected: setSelectedEntities
         },
-        childrenBefore: ({ entities, columnSettings }) => div({ style: { display: 'flex', alignItems: 'center', flex: 'none' } },
+        childrenBefore: ({ entities, columnSettings, showColumnSettingsModal }) => div({ style: { display: 'flex', alignItems: 'center', flex: 'none' } },
```

Comment:
Does `showColumnSettingsModal` reference the const in `DataTable`? I'm having trouble figuring out how it works.

Reply:
Yes. `EntitiesContent` passes `DataTable` a `childrenBefore` render prop that renders the menu above the table. `DataTable` passes `showColumnSettings` and a few other things to that render prop.

Reply:
Thanks!